### PR TITLE
Documentation - Add spaces between array elements coding guideline

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Title

### CODE REVIEW

File: docs/wiki/development/coding-guidelines.md [hunk 1]
```diff
@@ -397,6 +397,23 @@ Magic numbers are any of the following:
 
 [Source](http://en.wikipedia.org/wiki/Magic_number_%28programming%29){:target="_blank"}
 
+### 5.7 Spaces between array elements
+When using array notation `[]`, always use a space between elements to improve code readability.
+
+Good:
+
+```js
```
Comment (synixebrett) [inline docs/wiki/development/coding-guidelines.md]: does github not have sqf formatting?

Comment (jonpas) [reply docs/wiki/development/coding-guidelines.md]: No, documentation uses Jekyll which last I checked does not support SQF.

Comment (synixebrett) [reply docs/wiki/development/coding-guidelines.md]: I believe it uses [rouge-ruby/rouge](https://github.com/rouge-ruby/rouge) which [claims it does](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers)

Comment (jonpas) [reply docs/wiki/development/coding-guidelines.md]: Not with our configuration. @bux ?

**Comment by commy2** [comment]
nooooooooo

File: docs/wiki/development/coding-guidelines.md (hunk 1 above)
Comment (veteran29) [reply docs/wiki/development/coding-guidelines.md]: If you're using jekyll on GH pages I bet it's becouse of them using older version of rogue. You can't control their dependencies.

**Pull request merged**